en_US.json

```diff
@@ -1420,14 +1422,17 @@
 	"page.signUpGooglePlus": "Sign up with Google Plus",
 	"page.signUpTermsConditions": "I agree with terms and condtions",
 	"page.signUpTitle": "Isomorphic",
-	"page.title.certificate.list": "Certificates",
+	"page.title.cacertificate.list":"CA Certificates",
+	"page.title.certificate.list": "Server Certificates",
 	"page.title.contact.list": "Users",
 	"page.title.edgeHostname.list": "Edge Hostnames",
 	"page.title.property.list": "Properties",
 	"page.title.role.list": "Roles",
 	"page.title.task.deployment": "Deployment Task Detail",
 	"page.title.task.list": "Tasks",
 	"page.title.task.validation": "Validation Task Detail",
+	"page.tooltip.cacertificate.list":"Trusted CA certificates to be used to verify client certificates. Upload certificates here and reference them in TLS Settings of your properties.",
+	"page.tooltip.certificate.list": "Certificates to be used for SSL handshakes between client and CDN servers. Upload certificates here and reference them in TLS Settings of your properties. If your origin enforces authentication via certificates, please also upload certificates here, then reference them in the origin Advanced Settings of your properties.",
```

Comment:
in the second sentence, I'd insert the word "and"     to make  "please also upload certificates here, and then reference them in the origin Advanced Settings..."

Reply:
Thanks. Will add the word "and" as per suggestion.